gitapp/lib/users_list/connection_presenter.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:connectivity/connectivity.dart';
+
+class Connection {
+  static final Connection instance= new Connection();
+  Future<bool> CheckConnection() async {
+    ConnectivityResult connectivityResult =
+        await (Connectivity().checkConnectivity());
+
+    if (connectivityResult == ConnectivityResult.mobile) {
```

Comment:
Maybe we could merge those two return cases into one ?

Reply:
solved